cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3333"
+}
```

Comment:
Can this file live in `cypress/` somehow? Grouping the cypress-related things in the cypress directory would make it more self-contained and nice IMO :)

Reply:
Yes no worries! :grinning: Moved in f85c9eaef. Cypress does this annoying thing where `--config-file` has to be at the *end* of the command. So instead of making it nice and general by having it in the `cypress` script in `package.json` we have to add it wherever it's run. Added it to `test-e2e` and then we can add it to all newer commands we add later.